zerver/views/registration.py

```diff
@@ -359,28 +385,63 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
-                 # password_auth_enabled is normally set via our context processor,
-                 # but for the registration form, there is no logged in user yet, so
-                 # we have to set it here.
-                 'creating_new_team': realm_creation,
-                 'password_required': password_auth_enabled(realm) and password_required,
-                 'require_ldap_password': require_ldap_password,
-                 'password_auth_enabled': password_auth_enabled(realm),
-                 'root_domain_available': is_root_domain_available(),
-                 'default_stream_groups': get_default_stream_groups(realm),
-                 'accounts': get_accounts_for_email(email),
-                 'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
-                 'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
-                 'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
-                 'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
-                 }
+        context = {
+            'form': form,
+            'email': email,
+            'key': key,
+            'full_name': request.session.get('authenticated_full_name', None),
+            'lock_name': name_validated and name_changes_disabled(realm),
+            # password_auth_enabled is normally set via our context processor,
+            # but for the registration form, there is no logged in user yet, so
+            # we have to set it here.
+            'creating_new_team': realm_creation,
+            'billing_enabled': settings.BILLING_ENABLED,
+            'password_required': password_auth_enabled(realm) and password_required,
+            'require_ldap_password': require_ldap_password,
+            'password_auth_enabled': password_auth_enabled(realm),
+            'root_domain_available': is_root_domain_available(),
+            'default_stream_groups': get_default_stream_groups(realm),
+            'accounts': get_accounts_for_email(email),
+            'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
+            'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
+            'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
+            'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
+        }
     )
 
+def send_email_to_support(realm_name: str, organization_type: str,
+                          repository_link: str,
+                          organization_nature: str,
+                          organization_website: str,
+                          realm_link: str) -> None:
+    if organization_type == "Open-source":
+        send_email(
+            "zerver/emails/organization_info",
+            to_emails=[FromAddress.SUPPORT],
+            from_name="Zulip Account Security",
+            from_address=FromAddress.tokenized_no_reply_address(),
+            context={
+                'realm_name': realm_name,
```

Comment:
I think it'd be better to pass the `realm` object into this function and then get that via `zulip_default_context`.

Reply:
@timabbott I don't think we can use the name from `zulip_default_context`, because it gets `realm_name` from the request, but realm_creation happens on the root subdomain - so this won't give us the target realm?

Reply:
Ahh, indeed, we could as a prep commit extract from the `common_context` function a helper `common_context_unauthed` that has everything but the `user_name` part and accepts a `Realm` as its argument.  I think that'd be the best way to do this.